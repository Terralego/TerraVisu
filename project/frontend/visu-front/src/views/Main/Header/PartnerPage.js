import React from 'react';
import { Card } from '@blueprintjs/core';


const PartnerPage = () => (
  <Card>
    <div className="mentions-legales">
      <h1>Put your partner data here</h1>
    </div>
  </Card>
);

export default PartnerPage;
