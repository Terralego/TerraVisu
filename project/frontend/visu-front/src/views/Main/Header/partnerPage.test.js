import React from 'react';
import renderer from 'react-test-renderer';
import PartnerPage from './PartnerPage';

it('should render correctly', () => {
  const tree = renderer.create(
    <PartnerPage />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
