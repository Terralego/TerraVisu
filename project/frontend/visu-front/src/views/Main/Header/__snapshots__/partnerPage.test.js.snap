// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="bp3-card bp3-elevation-0"
>
  <div
    className="mentions-legales"
  >
    <h1>
      Put your partner data here
    </h1>
  </div>
</div>
`;
